distrobuilder/main.go

```diff
@@ -704,17 +701,11 @@ EOF
 }
 
 ## Main logic
-# Exit immediately if not a LXC/LXD container or VM
-if ! is_lxd_vm && ! is_lxc_container; then
-	exit
-fi
+# Nothing to do in LXD VM but deployed in case it is later converted to a container
+is_lxd_vm && exit 0
 
-# Check for NetworkManager and cloud-init
-nm_exists=0
-cloudinit_exists=0
```

Comment:
You're removing those in the first commit but then fixing up the codepaths in the second commit.
Would be cleaner if the first commit was just adding the VM check + re-indent and second commit would cleanup those two variables.